python/frontend/cirrus/cirrus/lambdas/deploy.sh

```diff
@@ -0,0 +1,5 @@
+rm bundle.zip
+zip -9r bundle.zip handler.py mmh3* min_max_helper.py ../utils.py normal_helper.py redis/ rediscluster/ toml.py redis.toml feature_hashing_helper.py
```

Comment:
These folders redis/, rediscluster/, etc.. need to be setup in this script.

Reply:
Created a separate script called `setup.sh`